docs/enhancements/arbiter-clusters.md

```diff
@@ -0,0 +1,144 @@
+---
+title: arbiter-clusters
+authors:
+  - "@giladravid16"
+creation-date: 2025-03-19
+last-updated: 2025-03-19
+---
+
+# Support 2 Node + 1 Arbiter Node HA Cluster
+
+## Summary
+
+This enhancement describes adding support in assisted-service to install TNA
+(Two Nodes + Arbiter) clusters - OpenShift clusters with control planes that
+consists of at minimum 2 normal-sized nodes, and at least 1 node that can be
+less powerful than the recommended node size.
+Arbiter nodes will only be running critical components for maintaining 
+HA (e.g etcd) to allow the arbiter node size to be as small and as low cost as
+possible within reason.
+
+## Motivation
+
+Customers at the edge are requiring a more economical solution for HA deployments
+at the edge. They can support running 2 node clusters for redundancy but would like 
+the option to deploy a lower cost node as an arbiter to supply the 3 nodes for etcd quorum.
+
+### Goals
+
+- Allow for a TNA cluster to be installed.
+- Allow adding arbiter nodes as day2 nodes to TNA clusters only.
+
+### Non-Goals
+
+- Moving from 2 + 1 to a conventional 3 node cluster.
+
+## Proposal
+
+This enhancement will include the following:
+- In the API we will:
+  - Add a new host_role for arbiter.
+  - Add a new field for clusters named arbiter_count. The default value will
+    be zero, but if it's set to a higher amount then the cluster will be
+    considered a TNA cluster.
+  - Add more values to the enums for cluster's hyperthreading and disk
+    encryptions in order to specify that they should be enabled on arbiters.
+  - Add a new feature_support_level_id for TNA.
+- We will update the clusters' validations to support TNA clusters.
+- We will update the clusters' transition handler to consider arbiter nodes.
+- We will add arbiter nodes' minimum requirements for clusters and per operator.
+- We will auto-assign the arbiter role to hosts until we have enough based
+  on the arbiter_count.
+
+### User Stories
+
+N/A - TNA clusters a future feature in openshift, the
+[openshift enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/arbiter-clusters.md#user-stories)
+has user stories that are not specific for assisted-service.
+
+### Implementation Details/Notes/Constraints
+
+The enums for cluster's hyperthreading and disk encryption will be:
```

Comment:
During kickoff it was raised that we may want to evaluate the usage of the disk encryption feature, but I'm not sure that really affects what we want to handle in this feature.